### PR TITLE
remove redundant code line

### PR DESCRIPTION
**Pull request by lstein**
install.bat was copying the requirements file into the install folder twice, causing an error message on the second try. This fixes the issue.

### CODE REVIEW

**Comment by ryanccn** [comment]
Will we have to manually apply these fixes to the `update.sh` if we already have an InvokeAI install?

File: installer/create_installer.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@
 cd "$(dirname "$0")"
 
 VERSION=$(grep ^VERSION ../setup.py | awk '{ print $3 }' | sed "s/'//g" )
-VERSION="$VERSION-p1"
+VERSION="$VERSION-p5"
```
Comment (mauwii) [inline installer/create_installer.sh:6]: since there is already a p5 on the release page, shouldn't this be p6 then 🤔

Comment (mauwii) [reply installer/create_installer.sh:6]: Tested succesful on Osx-arm64 and Win11-arm64 👍🏻

**Review by mauwii — APPROVED** [review]

**Comment by lstein** [comment]
> Will we have to manually apply these fixes to the `update.sh` if we already have an InvokeAI install?

I'll issue a downloadable script before the next release.

**Comment by ryanccn** [comment]
> > Will we have to manually apply these fixes to the `update.sh` if we already have an InvokeAI install?
> 
> I'll issue a downloadable script before the next release.

Okay, thanks!

**Pull request merged**